RELEASE-NOTES.txt

```diff
@@ -1,10 +1,17 @@
-3.0
+2.11
 -----
- 
+
+2.10
+-----
+* Added improvements for devices with hardware keyboards
+* Added support for screen readers when adding a new note
+* Fixed an issue with font size resizing when swiping
+* Added condensed list option for widget
```

Comment:
I added these manually because, as the diff shows, there was nothin here after the 2.9 notes.

If these are not good, feel free to suggest an edit via the GitHub interface and I'll take it from there to generate the GlotPress updates.